### ISSUE TITLE
User lookup is slow, needs full MAPI login

### ISSUE THREAD

**Issue by hoffie** [zarafagroupware/python-zarafa#29]
`zarafa.User("foo")` seems to be a very slow operation. When running with strace, one can see that some component (probably libmapi itself?) performs a full MAPI login request.
This seems to be caused by the [OpenEntry call](https://github.com/zarafagroupware/python-zarafa/blob/master/zarafa/__init__.py#L2632). The good thing is that this call is only necessary for the mapiobj retrieval, which itself is not needed in all cases.

Long story short, I am seeing big performance improvements (e.g. when outputting lots of messages' Address fields) when postponing the mapiobj retrieval until needed. Will submit a pull request in a minute.

**Comment by jelly** [comment]
Fixed by fb30464, thanks!
